editor/project_manager.h

```diff
@@ -55,46 +55,46 @@ class ProjectManager : public Control {
 
 	static ProjectManager *singleton;
```

Comment:
here we don't need `static ProjectManager *singleton = nullptr;` ?

Reply:
it is already in the c++ file as it is a static